src/main/kotlin/io/zerobase/smarttracing/features/models/ModelsResource.kt

```diff
@@ -1,18 +1,18 @@
-package io.zerobase.smarttracing.resources
+package io.zerobase.smarttracing.features.models
 
-import io.zerobase.smarttracing.MultiMap
+import com.google.common.collect.Multimap
```

Comment:
I didn't notice this before / that smart tracing had its own multi map; what exactly was the difference with the MultiMaps?

Reply:
the old one was a type-alias, the new one is a class that has a more convenient API